scripts/docker-command.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
```

Comment:
The build failure is due to the missing `LICENSE` header, could you add it?
```suggestion
# Copyright (c) Facebook, Inc. and its affiliates.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

```

Reply:
Oh yes, it was the header check, I was reading the wrong thing.

Anyway, I accepted your suggested commit. Thank you!